### PR TITLE
Update LOCALIZATION_CONTRIBUTORS

### PR DESCRIPTION
**Pull request by alejandro-medici**
* Update LOCALIZATION_CONTRIBUTORS
* Alejandro Medici for Spanish added

### CODE REVIEW

**Comment by clausia** [comment]
@alejandro-medici has successfully joined Crowdin

File: LOCALIZATION_CONTRIBUTORS [hunk 1]
```diff
@@ -80,6 +80,7 @@ John Michael Martinez Tenorio
 Pablo Vicente Greco
 Emanuel Dri (AR)
 Edgar Bruney Castañeda Torres
+Alejandro Medici (AR)
```
Comment (clausia) [inline LOCALIZATION_CONTRIBUTORS:83]: We do not include the countries where each person is from, please check the list of the other languages, in a later commit the "`(AR)`" of the contributor that also has the country will be removed.
Please remove the parentheses indicating your country.

**Review by clausia — CHANGES_REQUESTED** [review]

**Review by alejandro-medici — COMMENTED** [review]
Country removed

**Review by clausia — APPROVED** [review]
LGTM

**Pull request merged**